pymesync.py

```diff
@@ -535,6 +535,10 @@ def __local_auth_error(self):
 
     def __response_to_python(self, response):
         """Convert response to native python list of objects"""
+        # DELETE returns an empty body if successful
+        if not response.text and response.status_code == 200:
```

Comment:
@ElijahCaine It isn't only used for deletes, this if statement is what handles what TimeSync returns upon a successful DELETE. The rest of the method covers all responses. It takes a response and returns a python dict. The response doesn't only include the text field, but also status_code and some others I don't use.